modules/gem5/_posts/2000-03-01-assignment1.md

```diff
@@ -95,6 +84,8 @@ Before starting with simulations, answer the following questions in your report.
 
 1. What metrics should you use to measure the performance of a computer system? Why?
 2. Why is it not always possible to use the same metrics for performance to evaluate computer systems?
+3. Define the Iron law of processor performance.
+4. What does an Instruction Set Architecture (ISA) define?
```

Comment:
Let's drop this. It's a good question, but it's covered in the quizzes

Reply:
Sure. I wanted to put his question to refresh the students' memory on the Iron Law as this assignment is based on the same.

Reply:
Keep question 3, drop question 4 :)

Reply:
done.